docs/static/monitoring/monitoring-ea-dashboards.asciidoc

```diff
@@ -0,0 +1,109 @@
+[[dashboard-monitoring-with-elastic-agent]]
+=== Collect {ls} monitoring data for dashboards
+[subs="attributes"]
+++++
+<titleabbrev>Collect monitoring data for dashboards</titleabbrev>
+++++
+
+ToDo: Add brief intro describing the feature and value prop. 
+So far, we haven't differentiated between the two. 
+Add info to help them decide which one they need.  
+
+**Prerequisites**
+
+Complete these steps as you prepare to collect and ship monitoring data for dashboards:
+
+[[disable-default-db]]
+.Disable default collection of {ls} monitoring metrics
+[%collapsible]
+====
+include::monitoring-prereq-disable-default.asciidoc[]
+====
+
+[[define-cluster__uuid-db]]
+.Specify the target cluster_uuid (optional)
+[%collapsible]
+====
+include::monitoring-prereq-define-cluster.asciidoc[]
+====
+
+
+[discrete]
+[[install-and-configure-db]]
+=== Install and configure {agent}
+
+Install and configure {agent} to collect {ls} monitoring data for dashboards.
+We'll walk you through the process in these steps:
+
```

Comment:
What do you think about adding a link to the [general agent installation page](https://www.elastic.co/guide/en/fleet/current/elastic-agent-installation.html) somewhere, for more details than we will give here? I'm not sure if this is the right place, but it might work here, or at the bottom of this section?